src/docker.js

```diff
@@ -62,8 +62,14 @@ const getInterfaceAddress = (ifconfig) => {
 };
 
 const pullImage = async (docker, image) => {
-  const followProgress = promisify(docker.modem.followProgress);
-  await followProgress(await docker.pull(image));
+  const stream = await docker.pull(image);
+  await new Promise(async (resolve, reject) => {
+    docker.modem.followProgress(stream, resolve, (progress) => {
+      if (debugDocker) {
+        console.log(`${image}: ${progress.status} ${progress.progress ? progress.progress : ''}`);
```

Comment:
Doesn't this just print progress on pulling the image? Based on the docs ('To debug some docker logs...'), I would have likely guessed runtime docker container logs. `docker logs` is a docker CLI command which prints container logs, so making this assumption seems reasonable to someone looking at the docs only. Perhaps a tweak to the description would make it more clear.

Since this is the third (unless I missed some) place where we conditionally enable debug logging (log namespacing more or less), perhaps it would be a good time to switch to a tool that does just this rather than chaining a bunch of special purpose env vars. Perhaps [`debug`](https://github.com/visionmedia/debug) or some other similarly popular library.  Enabling logging would simply become `DEBUG=lambda,docker,localstack yarn test`

Not a blocker, but I think it'd simplify things a bit and set us up nicely for future additions. Especially if you think about having to forward all of these environment variables vs. just one (potentially multiple times if nesting containers).

Reply:
I like the idea of replacing all of the console logs in the project to instead use some other library.  I am thinking Bunyan would be appropriate here, as we already use it for LifeOmic and it formats logs nicely.  The only other part would be parsing the log level from the env, or allowing it to be set in the library.